proxmox/pools/pool_types.go

```diff
@@ -45,5 +47,12 @@ type PoolListResponseData struct {
 
 // PoolUpdateRequestBody contains the data for an pool update request.
 type PoolUpdateRequestBody struct {
+	// The pool's comment
 	Comment *string `json:"comment,omitempty" url:"comment,omitempty"`
+	// If this is set to 1, VMs and datastores will be removed from the pool instead of added.
+	Delete *types.CustomBool `json:"delete,omitempty" url:"delete,omitempty,int"`
+	// The list of virtual machines to add or delete.
+	VMs *[]string `json:"vms,omitempty" url:"vms,omitempty,comma"`
```

Comment:
We'd need a custom JSON serialization to a comma-separated list, though it works now as there is only a single VM ID in the use case.